src/platform_impl/windows/event_loop.rs

```diff
@@ -1217,7 +1217,18 @@ unsafe fn public_window_callback_inner<T: 'static>(
                 }
             }
 
-            0
+            // todo(msiglreith):
+            //   Ideally, `WM_SYSCHAR` shouldn't emit a `ReceivedChar` event
+            //   indicating user text input. As we lack dedicated support
+            //   accelerators/keybindings these events will be additionally
+            //   emitted for downstream users.
+            //   This means certain key combinations (ie Alt + Space) will
+            //   trigger the default system behavior **and** emit a char event.
```

Comment:
Would it make sense to mention that this is done in order to have consistent behavior across different platforms?

Viewed from the Windows only perspective, it is indeed strange.
But if a cross-platform application wants to react to `Alt+A`, unless `winit` emits the same sequence of events on all platforms for `Alt+A`, the application logic would need to become platform specific.
For applications that want to process `Alt+key` in a uniform manner, it becomes even more work, as they would need to repeat the decoding written above in this branch.